### DISCUSSION TITLE
Soft page breaks

### DISCUSSION THREAD

**Discussion by fizzyade** [outline/outline#2071] [Ideas]
This is from a conversation on Twitter with the team.

Something that I need control over is the creation of soft page breaks so that I have *some* control when outputting PDF or printing.  The "infinite" page is absolutely fine for viewing and editing on a device, but if you are putting together a technical note then you often want to be able to select where a new page starts.

On Twitter you pointed out that the difficulty is that Markdown is the format of the documents and that you weren't clear on how something like this could be achieved.

My proposal to you was that Markdown has 3 types of page breaks.

```
---
***
___
```

It's unlikely that people will be using a mix/match of those and most users will stick to a single type, this provides the hook to make my suggestion possible.

An option could be added to the software which allows you to configure the "type" of break that is being inserted, by default they would all be just a plain divider.

Now, in the configuration the user could select "***" and mark it as "Soft Page Break".

While viewing and editing the document in the software, the break/divier behaves just as normal, your document remains an "infinite length" page, nothing changes.

But, when it comes to sending the document to a PDF/printer, if the application comes across a divier type which is set to "soft page break", then a page break is inserted into the printed output and everything following the break appears on the next "page", rinse repeat!

- This "tweak" requires no special tags, it uses "plain old markdown".
-  This "tweak" breaks nothing, by default the document will look and behave the same on screen, even if the user has set all "dividers" to "soft page break".
-  This "tweak" only affects PDF/printed output, and it only changes the printed output if the user has set the break type for a divider style to "soft page break".
- Users will rejoice and carry you on their shoulders over the new feature that provides a level of control over the output of their documents.

I think this pretty much sums up and describes the functionality I am proposing.  This has the potential to make the output from the application even more useful.

**Comment by tommoor** [comment]
I went ahead and did the work for this, it should land on the cloud version of Outline in the next week – I'll ping here when it's deployed.

https://github.com/outline/rich-markdown-editor/pull/433

**Reply by fizzyade** [reply]
Thanks Tom!  Wow, I really wish I’d found Outline sooner.

Can’t wait to try it out, I think it’s a really useful feature to have and I’m sure other people will find it very useful, they just don’t know it’s missing! :p

**Reply by tommoor** [reply]
This feature is now live, thanks for the request. You can insert via the /slash menu or convert an existing divider by clicking on it and selecting "page break"
